docker/Dockerfile.ng

```diff
@@ -0,0 +1,213 @@
+FROM usualoma/ruby-with-therubyracer:2.4.0-alpine
```

Comment:
Can you remake this image based on ruby `2.3`? I think it would make this PR a bit lighter and separate it from the ruby upgrade in #1876. I would also like to see how much work is required to switch ruby versions in alpine as we will need to move to ruby `2.4.x` and `2.x` eventually.